src/Sidekick.Apis.Poe/Parser/Modifiers/ModifierParser.cs

```diff
@@ -102,15 +99,9 @@ private IEnumerable<ModifierMatch> MatchModifiers(ParsingItem parsingItem)
                     continue;
                 }
 
-                var patterns = MatchModifierPatterns(block, line, lineIndex).ToList();
-                if (patterns.Count == 0)
-                {
-                    // If we reach this point we have not found the modifier through traditional Regex means.
-                    // Text from the game sometimes differ from the text from the API. We do a fuzzy search here to find the most common text.
-                    patterns = [.. MatchModifierFuzzily(line)];
-                }
+                var patterns = GetPatterns(block, lineIndex, line, parsingItem);
 
-                if (patterns.Count is 0)
+                if (!patterns.Any())
```

Comment:
Any() is technically slower, but at this scale, I don't really mind. Though next time I work on this file, my IDE will put it back at .Count

Reply:
FYI, I use Rider

Reply:
I had no idea, I thought this was only for older versions of .NET.